src/project/python.rs

```diff
@@ -25,6 +26,22 @@ pub enum ParsePyPiRequirementError {
     MissingOperator(String),
 }
 
+impl fmt::Display for PyPiRequirement {
```

Comment:
I think its better to reuse the code in the function below this as that does the format too but only to put it into a Item. So if you pull out some of the "to string" work there and reuse this fmt there aswell that would be awesome!

Reply:
done

Reply:
Haha that also works